Neo4j.Driver/Neo4j.Driver/Experimental/IBookmarkManager.cs

```diff
@@ -0,0 +1,57 @@
+﻿// Copyright (c) 2002-2022 "Neo4j,"
+// Neo4j Sweden AB [http://neo4j.com]
+// 
+// This file is part of Neo4j.
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// 
+//     http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Neo4j.Driver.Experimental;
+
+/// <summary>
+/// Experimental: Subject to change.
+/// Manager of Neo4j's causal-consistency mechanism: <see cref="Bookmarks"/>.<br/>
+/// The manager maintains and provides collections of bookmarks for all databases,
```

Comment:
A single collection, no plural. Also not sure "databases" needs to be mentioned at all.